src/VisualStudio/Core/Def/Implementation/ExtractInterface/ExtractInterfaceDialog.xaml

```diff
@@ -122,6 +122,7 @@
                 </Grid.ColumnDefinitions>
                 <u:AutomationDelegatingListView x:Uid="MemberSelectionList"
                           x:Name="Members"
+                          AutomationProperties.Name="Members"
```

Comment:
Would this work? All new to me.

```suggestion
                          AutomationProperties.LabeledBy="{Binding ElementName=MemberSelectionGroupBox}"
```

📝 You would need to at least set the `x:Name` of the group box to use this approach.

Reply:
New to me as well, I'll give it a go! I've been using Keros to identify these issues.

Reply:
Passes tests, and probably even better than the solution I had. @jasonmalinowski  this prevents the need for localization as well (which I presume was needed before)